www/js/history/review-controller.js

```diff
@@ -2,13 +2,50 @@ angular.module('starter.review', [])
 
 .controller('ReviewCtrl', function($scope, $stateParams) {
 
+	$scope.newReview = {};
+
+// START: Five star rating thang
+	// Initialization code
+	var fiveStarElement = document.getElementById("fiveStarRating");
+	var fourStarElement = document.getElementById("fourStarRating");
+	var threeStarElement = document.getElementById("threeStarRating");
+	var twoStarElement = document.getElementById("twoStarRating");
+	var oneStarElement = document.getElementById("oneStarRating");
+
+	oneStarElement.classList.add('button-positive');
```

Comment:
Yeah Ariel and I spoke about that briefly. There's a card in the Trello ("Require star rating field...") that should cover this.